bin/picca_export.py

```diff
@@ -121,4 +121,4 @@
     h.write([rp,rt,z,da,co,dm,nb],names=['RP','RT','Z','DA','CO','DM','NB'],comment=comment,header=head,extname='COR')
     comment = ['R-parallel model','R-transverse model','Redshift model']
     h.write([dmrp,dmrt,dmz],names=['DMRP','DMRT','DMZ'],comment=comment,extname='DMATTRI')
-    h.close()
```

Comment:
h.close() has the wrong indentation. There should be no changes to picca_export.py